principles.md

```diff
@@ -3,12 +3,12 @@
 <br>
 
 ### Proposal Principles to Use to Achieve the Goal​
-- Boy Scout Rule​
+~~- Boy Scout Rule​~~ this is just a minor improvement (may risk of breaking things)
 - KISS​
 - YAGNI​
-- Keep things DRY​
+~~- Keep things DRY​~~ have to do this with special care, sometimes DRY could affect KISS
```

Comment:
Would be great if you could share some examples for your concerns.
From there, we could find ways to sort the "conflicts" out.

Reply:
anything in this [utilities](https://github.com/iPriceGroup/dalake/tree/develop/app/iprice/utilities) are highly DRY, no clear responsibility, but one day they will end up hard to maintain like [Sanitize.php](https://github.com/iPriceGroup/web/blob/master/library/util/Sanitize.php) on FE

Reply:
I am cool with this